cmd/notary/tuf.go

```diff
@@ -546,3 +587,30 @@ func getRemoteTrustServer(config *viper.Viper) string {
 	}
 	return defaultServerURL
 }
+
+func getTrustPinning(config *viper.Viper) (trustpinning.TrustPinConfig, error) {
+	var ok bool
+	// Need to parse out Certs section from config
+	certMap := config.GetStringMap("trust_pinning.certs")
+	resultCertMap := make(map[string][]string)
+	for gun, certSlice := range certMap {
+		var castedCertSlice []interface{}
+		if castedCertSlice, ok = certSlice.([]interface{}); !ok {
+			return trustpinning.TrustPinConfig{}, fmt.Errorf("invalid format for trust_pinning.certs")
```

Comment:
Just out of curiosity, is there a best practice in go about returning pointers vs objects? 

I went to this neat [go training meetup](https://github.com/ardanlabs/gotraining/) where the instructor mentioned that unless data needs to be modified that it's better to pass by value.  I think his reasoning was just optimizing for having the value be on the stack rather than the heap.  

It seems like premature optimization for us to really care about it that much, but I was just wondering if there's a counter-argument for passing pointers for just data other than being able to determine whether the user specified a zero value?

In this case we don't need to be able to determine the difference between default trust pinning (the zero value) and whether the user specified any trust pinning (for instance, if it matters whether we can tell the difference between an empty string or just an unspecified string).